releasenotes/notes/1.3/deprecate-pulse-package-07a621be1db7fa30.yaml

```diff
@@ -47,15 +46,15 @@ deprecations_transpiler:
       * :class:`~qiskit.transpiler.passes.RZXCalibrationBuilder`
       * :class:`~qiskit.transpiler.passes.EchoRZXWeylDecomposition`
   - |
-    The ``inst_map`` argument in :func:`~qiskit.transpiler.generate_preset_pass_manager`,
-    :meth:`~transpiler.target.Target.from_configuration`, :class:`~qiskit.transpiler.PassManagerConfig` initializer
+    The ``inst_map`` argument in :func:`.generate_preset_pass_manager`,
+    :meth:`~.Target.from_configuration`, :class:`~qiskit.transpiler.PassManagerConfig` initializer
     and :func:`~qiskit.transpiler.preset_passmanagers.common.generate_scheduling` is being deprecated.
   - |
     The ``calibration`` argument in :func:`~qiskit.transpiler.target.InstructionProperties` initializer methods is being
     deprecated.
 deprecations_visualization:
```

Comment:
Let's remove this section altogether. `pulse_drawer` is not part of the public API. Furthermore we're already mentioning in the general deprecation note that pulse visualization is also being deprecated and attempts to visualize pulse will raise deprecation warnings of pulse classes/builder functions anyway.

Reply:
done in 925f0e7